cs-amazon/src/main/java/io/cloudslang/content/amazon/factory/helpers/NetworkUtils.java

```diff
@@ -93,6 +96,17 @@
         return queryParamsMap;
     }
 
+    public Map<String, String> getCreateSubnetQueryParamsMap(InputsWrapper wrapper) {
+        Map<String, String> queryParamsMap = new LinkedHashMap<>();
```

Comment:
is it important here to use `LinkedHashMap` - if not, maybe consider `HashMap` which is slightly faster in general